src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
```

Comment:
```suggestion
    private static final int DEFAULT_INITIAL_CAPACITY = 16;
```
+0.5 techcheck points, but constants should be constants, not expressions